build.gradle

```diff
@@ -40,24 +40,24 @@ java {
 }
 
 wrapper {
-    gradleVersion = '8.10'
+    gradleVersion = '8.12'
 }
 
 dependencies {
     dependencies {
-        implementation "org.springdoc:springdoc-openapi-data-rest:${sprindocAnnotationsVersion}"
-        implementation "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
-        implementation "javax.validation:validation-api:${validationApiVersion}"
-        implementation "org.apache.commons:commons-lang3:${commonsLangVersion}"
+        api "org.springdoc:springdoc-openapi-starter-common:${springDocVersion}"
+        api "com.fasterxml.jackson.core:jackson-databind:${jacksonVersion}"
```

Comment:
We don't want our users rely on jackson version specified by our library, since we want to be free changing it. Please revert to `implementation`.

Reply:
fixed